aim2dat/strct/ext_manipulation/add_structure.py

```diff
@@ -108,11 +108,14 @@ def add_structure_coord(
     host_indices: Union[int, List[int]] = 0,
     guest_index: int = 0,
     guest_structure: Union[Structure, str] = "CH3",
+    guest_dir: List[float] = [0.0, 0.0, 0.0],
```

Comment:
We could choose `None` as default?

Reply:
I would leave it as `List[float] = [0.0, 0.0, 0.0]`, so the user sees immediately the input structure.

Reply:
If I may add my 2 cents, I'm also in favour of `None`. I think the format is quite clear from the docstring and `None` seems to be a more intuitive value since it will be overridden anyway.

Reply:
Also, we should avoid mutable types as input parameters in general. So if we keep [0,0,0], a tuple would be safer.

Reply:
Would you suggest to change it to the following then:
`guest_dir: Union[None, List[float]] = None`